src/content/structured/components/canary/date-input/guidance.mdx

```diff
@@ -0,0 +1,191 @@
+/>
+
+Refer to [typography styling](/styles/typography#text) for correct label, error text and helper text styles.
+
+#### Labels
+
+Use [content styling](/styles/content-style/principles) for help with building effective labels for people using date input.
+
+#### Helper text
+
+Use helper text to indicate restrictions in your date input such as not allowing weekends.
+
+#### Placeholder text
+
+Use a placeholder to demonstrate an example of the desired input. Do not add essential information into the placeholder as it will be hidden when a date is entered into the field.
+
+<DoubleDoDontCaution>
+  <DoDontCaution
+    imageSrc={figure4}
+    imageAlt="An example of the benefits of using helper text in a date input."
+    caption="Use helper text to give context to the current required input."
+  />
+  <DoDontCaution
+    imageSrc={figure5}
+    imageAlt="An example of a date input with no helper text included."
+    caption="Avoiding helper text removes people’s context."
+  />
+</DoubleDoDontCaution>
+
+## Date formatting
+
+Use the date input ‘dateFormat’ property to change the default input format between DD/MM/YYYY, MM/DD/YYYY and YYYY/MM/DD. Use differing labels on the helper text and within a date input, when catering for regional input preferences.
+
+## Interaction behaviour
+
+Use the ‘disablePast’ property when using date input to gather information about future dates, such as hotel bookings.
+
+Use the ‘ disableFuture’ property when using a date input to gather information about days in the past, such as birthdays.
+
+Set the default date on a date input to help people quickly complete the input by sticking with the default value. For example, set the default date to the next available delivery date when asking for a delivery date preference.
+
+Do not use default date when using date input to collect a date far in the past. For example, do not use “now” as default when asking for date of birth. Instead, use placeholder text of date format until the user selects their input.
+
+Use `html autofocus` to automatically focus a date input when it is the only input on a page.
+
+Do not use `html autofocus` on a date input when it is part of a form with other inputs.
+
+**You can copy and paste a complete date into the date input.**
+
+## Sizing
+
+Use the date input size properties to change its vertical size and achieve compact or spacious layouts.
+
+<DoubleDoDontCaution>
+  <DoDontCaution
+    imageSrc={figure6}
+    imageAlt="A date input matching the height of the button component next to it."
+    caption="Adjust a date input’s size properties to help it match the rest of your form."
+  />
+  <DoDontCaution
+    imageSrc={figure7}
+    imageAlt="A date input that is larger than the button component it is next to."
+    caption="Don’t mix different sized components."
+  />
+</DoubleDoDontCaution>
+
+### Width
+
+Ensure the width of the date input allows enough space for the full length of a person’s potential input.
+
```

Comment:
One for design, are both images meant to be the correct usage of the width of the date-input?
Seems strange having two 'correct' but differing images.